server/src/api.js

```diff
@@ -83,8 +92,15 @@ export function buildFastify(renderFn, compareFn) {
       const viewport = request.body.viewport;
       const fullpage = request.body.fullpage ?? false;
       const style = request.body.style;
+      const usehMhtmlConverter = request.body.usehMhtmlConverter ?? false;
 
-      const actual = await renderFn(actualHtml, viewport, fullpage, style);
+      const actual = await renderFn(
+        actualHtml,
+        viewport,
+        fullpage,
+        style,
+        usehMhtmlConverter,
```

Comment:
Here we should also change the naming to `actualMhtml` - but this would be a breaking change at API level.